designer/client/actions/nk/importExport.ts

```diff
@@ -0,0 +1,43 @@
+import HttpService from "../../http/HttpService"
+import {importProcess} from "./index"
+import {reportEvent} from "./reportEvent"
+import {withoutEmptyEdges} from "../../components/graph/GraphPartialsInTS/EdgeUtils";
+import {Process, ProcessId} from "../../types";
+import {ProcessVersionId} from "../../components/Process/types";
+import {ThunkAction} from "../reduxTypes";
+
+export function importFiles(files: File[], processId: ProcessId): ThunkAction {
+  return (dispatch) => {
+    files.forEach(
+      file => dispatch(importProcess(processId, file)),
+    )
+    return {
+      type: "IMPORT_FILES",
+    }
+  }
+}
+
+export function exportProcessToJSON(process: Process, versionId: ProcessVersionId): ThunkAction {
+  return (dispatch) => {
+    let noEmptyEdges = withoutEmptyEdges(process)
```

Comment:
Can we have this cleaning in one class used in both cases?

Reply:
added comment, changed name to suggest the need of better solution :)